recipe/build_dll.patch

```diff
@@ -0,0 +1,87 @@
+From 170f403ee2355d71dd6bca6ff463b7c341ad04dc Mon Sep 17 00:00:00 2001
+From: Serhii Kupriienko
+Date: Mon, 12 Feb 2024 15:55:52 +0200
+Subject: [PATCH] fix the dll creation, remove DESCRIPTION
+
+---
+ libbz2.def   |  1 -
+ makefile.msc | 30 ++++++++++++++++++------------
+ 2 files changed, 18 insertions(+), 13 deletions(-)
+
+diff --git a/libbz2.def b/libbz2.def
+index 2dc0dd8..b8de2aa 100644
+--- a/libbz2.def
++++ b/libbz2.def
+@@ -1,5 +1,4 @@
+ LIBRARY			LIBBZ2
+-DESCRIPTION		"libbzip2: library for data compression"

```

Comment:
Why is this removed?

Reply:
It's an outdated field, the compiler don't support it anymore